src/Ocelot/Configuration/Validator/FileValidationFailedError.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Ocelot.Errors;
+
+namespace Ocelot.Configuration.Validator
+{
+    public class FileValidationFailedError : Error
+    {
+        public FileValidationFailedError(string message) : base(message, OcelotErrorCode.UnknownError)
```

Comment:
Maybe the OcelotErrorCode should be OcelotErrorCode.FileValidationFailedError

Reply:
yes, it't ok to add a OcelotErrorCode named FileValidationFailedError

Reply:
and as we change all file validation error result to FileValidationFailedError , the unit test failed for 
 Ocelot.UnitTests.Configuration.FileConfigurationCreatorTests.should_return_validation_errors is ok and we just change PathTemplateDoesntStartWithForwardSlash to FileValidationFailedError